src/lib/components/CourseLinks/index.js

```diff
@@ -43,6 +43,22 @@ class CourseLinks extends HTMLElement {
     slot.addEventListener('slotchange', this.onSlotChange);
   }
 
+  /**
+   * Get the user's overall course progress from localstorage.
+   * This returns an object where each string key represents a different course.
+   * Example: {css: {pages: ['intro', ...], percent: 10}, pwa: {...}}
+   * @returns {Object.<string, Course>}
+   */
+  getProgress = () => {
+    let progress;
+    try {
+      progress = JSON.parse(localStorage['webdev_course_progress']);
```

Comment:
Since this is now in try/catch, you can just `return` this line and `return {}` below. But whatever 🤷